src/benchcab/utils/meorg.py

```diff
@@ -0,0 +1,96 @@
+"""Utility methods for interacting with the ME.org client."""
+from benchcab.internal import MEORG_CLIENT
+from meorg_client.client import Client as MeorgClient
+from hpcpy import get_client
+import benchcab.utils as bu
+import os
+from glob import glob
+
+def do_meorg(config: dict, upload_dir: str, benchcab_bin: str, benchcab_job_id: str):
+    """Perform the upload of model outputs to modelevaluation.org
+
+    Parameters
+    ----------
+    config : dict
+        The master config dictionary
+    upload_dir : str
+        Absolute path to the data dir for upload
+    benchcab_bin : str
+        Path to the benchcab bin, from which to infer the client bin
+
+    Returns
+    -------
+    bool
+        True if successful, False otherwise
+    """
+
+    logger = bu.get_logger()
+
+    model_output_id = config.get("fluxsite").get("meorg_model_output_id", False)
+    num_threads = MEORG_CLIENT["num_threads"]
+    
+    # Check if a model output id has been assigned
+    if model_output_id == False:
+        logger.info("No model_output_id found in fluxsite configuration.")
+        logger.info("NOT uploading to modelevaluation.org")
+        return False
+    
+    # Allow the user to specify an absolute path to the meorg bin in config
+    meorg_bin = config.get("meorg_bin", False)
```

Comment:
I didn't see `meorg_bin` in the config schema - can you update the schema and test that this works?

Reply:
That's because it isn't, I'll add that in.

Reply:
Done